ui/raidboss/data/03-hw/alliance/dun_scaith.js

```diff
@@ -131,7 +144,10 @@
       // Wailing Atomos is blue, Cursed Atomos is yellow.
       // 1C9F:Aether is the circle AoE, 1CA0:Aetherial Chakram is the donut AoE
       id: 'Dun Scaith Blue Atomos',
-      regex: / 14:(\y{AbilityCode}):\y{Name} starts using Juggling Sphere on Wailing Atomos/,
+      regex: / 14:1C(9F|A0):\y{Name} starts using Juggling Sphere on Wailing Atomos/,
+      regexDe: / 14:1C(9F|A0):\y{Name} starts using Jonglierball on Heul-Atomos/,
+      regexFr: / 14:1C(9F|A0):\y{Name} starts using Sphère Jongleuse on Gueule Gémissante/,
+      regexJa: / 14:1C(9F|A0):\y{Name} starts using ジャグリング・スフィア on 虚声のアトモス/,
```

Comment:
I believe this code, and the similar block following it will now be broken with the `matches[1]` below; we can change it into a matching group for the whole ability code or change the matches logic within the alertText to only validate on the two hex characters.

Reply:
changed the matching statements for both yellow and blue atomos